test/Microsoft.HttpRepl.Tests/Microsoft.HttpRepl.Tests.csproj

```diff
@@ -1,10 +1,32 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
I can't recall if it's necessary to do the remove or whether the SDK got smarter about this, but you could try to reduce duplication here by doing something like:
```
<ItemGroup>
    <EmbeddedResource Include="..." /> <!-- repeated for many files -->
    <None Remove="@(EmbeddedResource)" /> <!-- just remove any embedded resource from the None group -->
</ItemGroup>
```

Reply:
VS added the removes as I switched each file to Embedded resource, but they don't seem to be necessary as far as I can tell. I removed them and everything seems to work and not complain.